src/tests/unit/ingredients-serializer.test.js

```diff
@@ -0,0 +1,128 @@
+import chai from 'chai';
+import config from 'config';
+import _ from 'lodash';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+import { openapi } from 'utils/load-openapi';
+import { ingredientSerializerData } from './test-data';
+
+const ingredientAttributes = openapi.definitions.IngredientAttributes.properties;
+
+chai.should();
+
+let rawIngredient;
+let rawIngredients;
+let ingredientSerializer;
+
+const {
+  nullNotesIngredient,
+  emptyStringNotesIngredient,
+} = ingredientSerializerData;
+
+describe('test ingredients serializer', () => {
+  before(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
+    // see comment in doughs-serializer.test.js
+    ingredientSerializer = proxyquire('../../api/v1/serializers/ingredients-serializer', {});
+  });
+  after(() => {
+    sinon.restore();
+  });
+  describe('serializeIngredient', () => {
+    describe('when it gets an ingredient object with `notes` = null', () => {
+      beforeEach(() => {
+        rawIngredient = nullNotesIngredient;
+      });
+      it('changes null to empty string', () => {
+        const serializedIngredient = ingredientSerializer
```

Comment:
Some of the comments I left on the dough-serializer applies here as well. Especially looping tests. Look through your code and see where you can reduce code reuse. In fact it looks like some tests here are similar to the ones in dough-serializer. You can make a test-helper class to reuse test cases.